dask_expr/_backends.py

```diff
@@ -32,25 +33,31 @@
 )
 
 
+class ToPandasBackend(ToBackend):
+    @staticmethod
+    def operation(df, options):
+        from dask.dataframe.dispatch import to_pandas_dispatch
```

Comment:
any reason this is a local import? we import tons of stuff from dask.dataframe globally

Reply:
Nope - I'll move it to the top level